astroid/node_classes.py

```diff
@@ -1192,20 +1192,22 @@ def last_child(self):
 
 class Comprehension(NodeNG):
     """class representing a Comprehension node"""
-    _astroid_fields = ('target', 'iter', 'ifs')
+    _astroid_fields = ('target', 'iter', 'ifs', 'is_async')
```

Comment:
Let's put ``is_async`` in ``_other_fields``. ``_astroid_fields`` is only for objects which are AST objects.